docs/conf.py

```diff
@@ -15,6 +15,12 @@
 from os import listdir
 from os.path import isdir, join
 
+import django
+from django.conf import settings
+
+settings.configure()
```

Comment:
Can you explain a bit more about the exception and how this solves it?

Reply:
Sure, I added a comment with more information in https://github.com/open-telemetry/opentelemetry-python/pull/657/commits/5de0516af6b88b855165c623e63a20c036f0197e.